types/FingerTable.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import "fmt"
+
+type FingerEntry struct {
+	Id   []byte
+	GossiperAddress string
+}
```

Comment:
1. Could we type ID as a (hex) string? If the struct contains all fields that are "comparable" (slices are not comparable without writing custom loop in Go), you can *check structs for equality*, e.g,

`if fingerEntry1 == fingerEntry2 {...}`

There are many benefits to being able to compare directly structs, e.g., one can use those structs as keys in a map. Now having type `map[FingerEntry]whatever` is impossible because Finger Entries cannot be checked for equality, but if we get all fields of the struct directly comparable (by using directly comparable fields), we can create such a map. 

2. Do you mind keeping everything *unexported* (first letter lowercase, you probably know this) and export only if we *have* to because of the principle of least privilege and better system design principles. If we have everything unexported, we are more likely to think about how we can couple two modules in a less inter-dependent way.